src/concrete/ml/torch/lora.py

```diff
@@ -1,16 +1,22 @@
-"""This module contains classes for LoRA (Low-Rank Adaptation) training and custom layers."""
+"""This module contains classes for LoRA (Low-Rank Adaptation) FHE training and custom layers."""
 
 from typing import List, Tuple, Union
 
 import torch
+from torch import Tensor, nn
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+
+from .hybrid_backprop_linear import CustomLinear
+from .hybrid_model import HybridFHEModel
 
 try:
     from transformers import Conv1D as TransformerConv1D
-except ImportError:
+except ImportError:  # pragma: no cover
     TransformerConv1D = None
 
 # Create a tuple of linear layer classes to check against
-LINEAR_LAYERS: tuple = (torch.nn.Linear,)
+LINEAR_LAYERS: tuple = (nn.Linear,)
 if TransformerConv1D is not None:
```

Comment:
maybe:

LINEAR_LAYERS = (nn.Linear,) + ((TransformerConv1D,) if TransformerConv1D else ())

Reply:
I will keep the original code. Single line if else are pretty bad for proper coverage.